src/components/ViewOrder/OrderItem/OrderItem.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC } from "react";
+import { lightTheme } from "@deskpro/deskpro-ui";
+import {
+    H0,
+    P5,
+    P11,
+    Stack,
+    VerticalDivider,
+    HorizontalDivider,
+} from "@deskpro/app-sdk";
+import { TextBlockWithLabel } from "../../common";
+import img from "./item.png";
+import { OrderItemType } from "./types";
+
+const Image: FC<Partial<OrderItemType> & { basis?: number }> = ({ basis, name }) => (
+    <Stack basis={`${basis}%`}>
+        <img src={img} alt={name} style={{ width: "100%", height: "auto" }}/>
+    </Stack>
+);
+
+const NamePrice: FC<Partial<OrderItemType>> = ({ name, price, mos }) => (
+    <Stack justify="space-between" style={{ marginBottom: "8px" }}>
+        <Stack vertical>
+            <P5>{name}</P5>
+            <P11 style={{ color: lightTheme.colors.grey80 }}>{mos} mos</P11>
```

Comment:
You must use the Deskpro theme hook available from the SDK. If there's a theme change and/or dark mode added in future, we should inherit from the wider Deskpro product:

```typescript
import { useDeskproAppTheme } from "@deskpro/app-sdk";

export const Example = () => {
  const { theme } = useDeskproAppTheme();

  // ...
};
```

Reply:
You'll need to do this in all components using `lightTheme` directly, then remove all `import { lightTheme } from "@deskpro/deskpro-ui";` imports.

Reply:
got it. I'll replace it  in all components

Reply:
done